flake.lock

```diff
@@ -15,22 +15,94 @@
         "type": "github"
       }
     },
-    "nixpkgs": {
+    "gomod2nix": {
+      "inputs": {
+        "nixpkgs": "nixpkgs",
+        "utils": "utils"
+      },
+      "locked": {
+        "lastModified": 1641803356,
+        "narHash": "sha256-ZjF43cckJLJCV9I+SXwTGNQGI52KrYwUhLt/R83TyKk=",
+        "owner": "tweag",
+        "repo": "gomod2nix",
+        "rev": "fc5adcc53031c5a3934500a57896a840a2bd073d",
+        "type": "github"
+      },
+      "original": {
+        "owner": "tweag",
+        "repo": "gomod2nix",
+        "type": "github"
+      }
+    },
+    "nix-bundle-exe": {
       "flake": false,
       "locked": {
-        "narHash": "sha256-LjxfZmqGhA3ZBxDBJyl0YELvfEtLDLaCbbnAXTtsGRc=",
-        "path": "./nix",
-        "type": "path"
+        "lastModified": 1649122558,
+        "narHash": "sha256-zvwFPFWoQusuYDjAYwMZUlwIlv5oD64fOz8v2gdjd3U=",
+        "owner": "3noch",
+        "repo": "nix-bundle-exe",
+        "rev": "65f46eed8603f0ba36f9b142521a53b69a10e561",
+        "type": "github"
+      },
+      "original": {
+        "owner": "3noch",
+        "repo": "nix-bundle-exe",
+        "type": "github"
+      }
+    },
+    "nixpkgs": {
+      "locked": {
+        "lastModified": 1625191069,
+        "narHash": "sha256-M8/UH9pMgQEtuzY9bFwklYw8hx0pOKtUTyQC8E2UTHY=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "d8079260a3028ae3221d7a5467443ee3a9edd2b8",
+        "type": "github"
+      },
+      "original": {
+        "owner": "NixOS",
+        "ref": "nixos-unstable",
+        "repo": "nixpkgs",
+        "type": "github"
+      }
+    },
+    "nixpkgs_2": {
```

Comment:
why are two sets of nixpkgs needed?

Reply:
I guess need to override nixpkgs in gomod2nix

Reply:
fixed